pages/dmp_knowledge_base/difficult_faq.md

```diff
@@ -18,10 +18,62 @@ toc: true
 - The project will likely have working documents. How are they organized, named, stored, backed-up and exchanged between collaboration partners?
 - Is there ethical questions or Intellectual Property Rights (IPR) to be considered and documented?
 - Text is data, images are data… Specific for the humanities: [10.7486/DRI.tq582c863](https://doi.org/10.7486/DRI.tq582c863)
-- Text
+
+### Who will evaluate my DMP?
+- [Horizon Europe - briefing slides for evaluators](https://ec.europa.eu/info/funding-tenders/opportunities/docs/2021-2027/experts/standard-briefing-slides-for-experts_he_en.pdf)
+- [Horizon Europe - video on evaluation of Open Science practices](https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/support/videos), [YouTube-link](https://www.youtube.com/watch?v=EiJ8RaD3WBw)
+
+<!--- ### Who can quality-control my DMP? -->
+
+### Where can a DMP be published?
+- [Zenodo](https://zenodo.org/) is an agnostic sharing platform which allows you to share files such as PDFs. Zenodo also allows versioning and will create a DOI for your files. [GitHub](https://github.com/) is another option, and may be connected with Zenodo to generate a DOI for your repository
+
+### I feel it is too much work to prepare my data for archiving in a domain specific archive
+
+Subject specific repositories such as [Metabolights](https://www.ebi.ac.uk/metabolights/editor/guides/Quick_start_Guide/Quick_start_overview), [PDB](https://www.wwpdb.org/deposition/tutorial) and [Sikt](https://sikt.no/tjenester/arkivere-data/klargjore-forskningsdata-arkivering) have guides/help-pages which makes it easier for you to prepare your data before submitting them to the repository. See if the repository you are submitting to has such a guide. Familiarizing yourself with such a guide early on, such as when drafting up a DMP, makes it easier for you to plan which metadata needs to be gathered in order for you to be able to submit to your repository of choice. <!--- Apart from Sikt, these are very geared towards life sciences. If anyone has better examples, please feel free to remove/replace/add  -->
+
+### What format should I store my research data in?
```

Comment:
Does not really fit in this section? (see also additional information in https://dhp-stottepakke.github.io/pages/1_data_description)